doc/src/site/sphinx/deployment/rsparkling_on_windows.rst

```diff
@@ -44,28 +43,6 @@ The script will check that you can:
 Troubleshooting
 ~~~~~~~~~~~~~~~
 
--  Error from running ``spark_connect``
```

Comment:
Can't happen anymore, we package the sparkling water jar inside the rsparkling, we are not downloading it anymore